src/nl/surf/eduhub_rio_mapper/ooapi/program.clj

```diff
@@ -78,9 +78,15 @@
 
 ;; must have at least one rio consumer
 (s/def ::Program/consumers
-  (s/cat :head (s/* ::common/consumer)
-         :rio ::Program/rio-consumer
-         :tail (s/* ::common/consumer)))
+  (s/with-gen
+    (s/and
+      not-empty
```

Comment:
Zet hier ajb een comment bij dat die not-empty er alleen in zit om foutmelding te verbeteren (en daarom ook de with-gen)